tests/test_collections.py

```diff
@@ -128,3 +138,446 @@ def test_add_to_collection(
+            driver, provider=collection_provider
+        )
+        accepted_page.goto()
+        assert CollectionModerationAcceptedPage(driver, verify=True)
+        accepted_page.loading_indicator.here_then_gone()
+
+        # Find the card for the project that was just accepted and click the link for
+        # this project to go to the Project Overview Page. It should be the first one
+        # in the table since the default sort order is by Date (newest first).
+        accepted_card = accepted_page.get_submission_card(collection_project.id)
+        accepted_card.find_element_by_css_selector(
+            '[data-test-submission-card-title]'
+        ).click()
+
+        # On the Project Overview Page, verify that the Collection Container indicates
+        # that the project has been included in the collection.
+        project_page = ProjectPage(driver, verify=True)
+        assert project_page.collections_container.present()
+        assert (
+            "Included in Selenium Testing Collection's Collection"
+            in project_page.pending_collection_display.text
+        )
+
+    def test_pre_moderation_collection_reject(
+        self, session, driver, must_be_logged_in, collection_project
+    ):
+        """Test the rejection of a project submission to a public branded collection
+        with a pre-moderation workflow.  In this test a project is submitted to a
+        collection and a collection moderator will 'reject' the submission and the
+        project is not included in the collection.  NOTE: In this test case User One is
+        used to login to OSF and must therefore be setup through the admin app as a
+        moderator or admin for the collection provider being used.  The test will use
+        the OSF api to create a submitted project using the session credentials of User
+        Two so that the project submitter and moderator are different users.
+        """
+
+        try:
+            # Get data for the Selenium Pre-moderated Collection
+            collection_provider = osf_api.get_provider(
+                session, type='collections', provider_id='selenium'
+            )
+
+            # Submit the project to the Selenium Pre-moderated Collection
+            self.submit_to_moderated_collection(
+                collection_provider, collection_project.id
+            )
+
+            # Navigate to the Collections Moderation Pending Page. NOTE: User must be a
+            # Moderator or Admin for the collection.
+            pending_page = CollectionModerationPendingPage(
+                driver, provider=collection_provider
+            )
+            pending_page.goto()
+            assert CollectionModerationPendingPage(driver, verify=True)
+            pending_page.loading_indicator.here_then_gone()
+
+            # Get the card for the project that was just submitted to the collection
+            submisison_card = pending_page.get_submission_card(collection_project.id)
```

Comment:
As discussed, typo, 6 occurrences in total

Reply:
Fixed.